README.md

```diff
@@ -88,6 +88,10 @@ numberOfReviewers: 0
 
 >Note: Number of reviewers has currently no impact on Github teams and all teams will be added as reviewers.
 
+>Note: This feature is not enabled in the hosted app below.  
+https://github.com/apps/auto-assign  
+To enable the team assignment, you need to host your own app and change the permission that the app can read the organization's team.  
+
```

Comment:
```suggestion

- Repository permissions
    - `Administration` - `Read-Only`
- Organization permissions
    - `Members` - `Read-Only`
```

I propose a change to better clarify permissions to be added.
https://github.com/kentaro-m/auto-assign/issues/138#issuecomment-736012032

Reply:
@kentaro-m Thank you very much for reviewing!
I added the permission that is supposed to be added.
7484ff1a7b2e194fd5a9858fd59e1ff5209a5a2d